01_fib/harirakr/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func fib(n int64) []int64 {
+	if n <= -93 || n >= 93 {
+		fmt.Fprintln(out, "fib: Range out of bounds; takes input > -93 and < 93")
+		return nil
+	}
+
+	var sign int64 = 1
+	if n < 0 {
+		sign = -1
+	}
+
+	var a, b, i int64
```

Comment:
Wouldn't bother declaring any of these, the following works just fine
```go
a, b := 0, 1

...

for i := 0; i < n * sign; i++ {
        ...
}
```

Reply:
Sure, will keep this in mind. Thanks